paper-checkbox.html

```diff
@@ -171,6 +172,7 @@
         vertical-align: middle;
         padding-left: var(--paper-checkbox-label-spacing, 8px);
         white-space: normal;
+        line-height: normal;
```

Comment:
Did you test that if the user sets a different line height on the host (say `line-height: 2`), this doesn't break it? I.e. Something like this: http://jsbin.com/vuyosa/edit?html,output